### DISCUSSION TITLE
Memmap preferences format

### DISCUSSION THREAD

**Discussion by Jayant1234** [lucidrains/self-rewarding-lm-pytorch#22] [Q&A]
Are the preferences sequence in memmap files supposed to contain the prompt as well? i.e prompt + pref_1 , prompt+pref_2? 
While another file tells the prompt length so that we can glean prompt from the preferences?
 OR 
 prompts are stored somewhere else?

**Comment by lucidrains** [comment]
they both contain the prompt, and the `prompt_len` is also stored

**Reply by Jayant1234** [reply]
Thanks for the clarification! Also huge respect for making this Repo!
